### DISCUSSION TITLE
test/testing-in-project/

### DISCUSSION THREAD

**Discussion by giscus** [yeopto/yeopto.github.io#16] [Comments]
# test/testing-in-project/

우리 프로젝트의 테스팅 전략을 세워보자(storybook, react-testing-library, msw)

https://yeopto.github.io/test/testing-in-project/

**Comment by Gilpop8663** [comment]
하루 스터디의 테스팅 도구 선정에 대한 글 잘 보았습니다 도움이 되었어요 🔥

**Comment by kyw0716** [comment]
제가 아는 그 센트가 아닌가봐요
